bookkeeper-server/src/test/java/org/apache/bookkeeper/auth/TestAuth.java

```diff
@@ -34,7 +34,6 @@
 import org.apache.bookkeeper.conf.ClientConfiguration;
```

Comment:
As identified test cases are missing. Adding few more test cases to the list -

1. Multiple bookkeeper clients (a few with ssl enabled with multiple bookies (all ssl enabled)
2. Multiple bookkeeper clients (a few with ssl enabled with multiple bookies (a few with ssl disabled)